binding.gyp

```diff
@@ -47,6 +47,7 @@
               'xcode_settings': {
                 'GCC_ENABLE_CPP_EXCEPTIONS': 'YES',
                 'MACOSX_DEPLOYMENT_TARGET': '10.15',
+                'CLANG_CXX_LANGUAGE_STANDARD': 'gnu++17',
```

Comment:
```suggestion
                'CLANG_CXX_LANGUAGE_STANDARD': 'c++17',
```

Reply:
Was in doubt which one to take. First had `c++17` but then changed to `gnu++17` as it is the default for nodejs: https://github.com/nodejs/node/blob/main/common.gypi#L546